crates/sui-indexer/src/models/objects.rs

```diff
@@ -5,39 +5,51 @@ use crate::models::owners::OwnerType;
 use crate::schema::objects;
 use diesel::prelude::*;
 use sui_json_rpc_types::SuiObjectData;
+use sui_types::base_types::EpochId;
+use sui_types::messages_checkpoint::CheckpointSequenceNumber;
 use sui_types::object::Owner;
 
 #[derive(Queryable, Insertable, Debug, Identifiable, Clone)]
 #[diesel(primary_key(object_id))]
 pub struct Object {
-    #[diesel(deserialize_as = i64)]
-    pub id: Option<i64>,
+    pub epoch: i64,
+    pub checkpoint: i64,
```

Comment:
Nit: Let's add some comments like this is the epoch/checkpoint in which the (object_id, version) gets updated?